CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contributing
+
+## Prerequisites
+
+The prerequesites for contributing to this repo are the same is if you were developing with it.
+
+- A good code editor.
+- [Node.js v12.13.0 and NPM 6.14.5](https://nodejs.org/en/download/)
+- [Git 2.25](https://git-scm.com/downloads)
+
```

Comment:
Perhaps a heading goes here to separate the prerequisites from the instructions?

Reply:
Added, and committed, although I'm not sure why it doesn't reflect here. Unless it is because the heading I added is outside of the code block you selected.

Reply:
Yeah, I think it uses the line I selected as the last line to show.  Looks good in the commit though.